src/test/java/Blackjack/domain/participant/PlayerTest.java

```diff
@@ -0,0 +1,105 @@
+package Blackjack.domain.participant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import Blackjack.domain.Bet;
+import Blackjack.domain.card.Card;
+import Blackjack.domain.card.Rank;
+import Blackjack.domain.card.Suit;
+import Blackjack.domain.game.GameStatus;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Function;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class PlayerTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"TWO:true", "TWO,NINE,JACK:true", "JACK,QUEEN,KING:false"}, delimiterString = ":")
+    @DisplayName("플레이어 카드 추가 가능 여부 판단 기능 테스트")
+    void playerAbleToAddCardTest(String rankNames, boolean expected) {
+        // given
+        List<Rank> playerRanks = createRanks(rankNames);
+        Player player = createParticipant(playerRanks, name -> new Player(name, Bet.startingBet()));
+        // when & then
+        assertEquals(expected, player.ableToAddCard());
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"TWO,FIVE:BLACKJACK", "QUEEN,ACE:TIE", "QUEEN,NINE,TWO:BLACKJACK"}, delimiterString = ":")
+    @DisplayName("Blackjack인 플레이어가 있는 GameStatus 계산 기능 테스트")
+    void determineGameStatusWithBlackjackTest(String dealerRankNames, String playerStatus) {
+        // given
+        List<Rank> playerRanks = List.of(Rank.JACK, Rank.ACE);
+        Player player = createParticipant(playerRanks, name -> new Player(name, Bet.startingBet()));
+        List<Rank> dealerRanks = createRanks(dealerRankNames);
+        Dealer dealer = createParticipant(dealerRanks, Dealer::new);
+        // when
+        GameStatus actual = player.determineGameStatus(dealer);
+        // then
+        assertThat(actual).isEqualTo(GameStatus.valueOf(playerStatus));
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"TWO:LOSE", "JACK,QUEEN,KING:LOSE"}, delimiterString = ":")
+    @DisplayName("Bust인 플레이어가 있는 GameStatus 계산 기능 테스트")
+    void determineGameStatusWithBustTest(String dealerRankNames, String playerStatus) {
+        // given
```

Comment:
playerStatus 바로 Enum으로 받을 수 있지 않나요? 

```suggestion
    @DisplayName("Bust인 플레이어가 있는 GameStatus 계산 기능 테스트")
    void determineGameStatusWithBustTest(String dealerRankNames, GameStatus playerStatus) {
        // given
```

Reply:
테스트 매개변수는 기본형만 지원하는 줄 알았는데 JUnit Jupiter에 암묵적 형 변환 기능이 내장되어 있어 사용자 인터페이스, Locale 및  LocalDate, LocalTime 등, 파일  및  경로, URL과 URI, Enum  하위 클래스 또한 지원한다는 것을 알게 되었습니다. 짚어주셔서 감사합니다!